src/components/runtime-common.js

```diff
@@ -123,6 +123,19 @@ export const SaveFilesHelp = () => {
   ])
 }
 
+export const SaveFilesHelpRStudio = () => {
+  return h(Fragment, [
+    p([
+      'If you want to save some files permanently, such as input data, analysis outputs, or installed packages, ',
+      h(Link, {
+        href: 'https://support.terra.bio/hc/en-us/articles/360026639112',
+        ...Utils.newTabLinkProps
+      }, ['move them to the workspace bucket.'])
+    ]),
+    p(['Note: Unlike Jupyter noteooks, RStudio files are not autosaved to workspace bucket. Autosaving *.Rmd files will be supported soon. Stay tuned.'])
```

Comment:
Typo `noteooks`. Also maybe run this text by @Alessandro (does he have a GitHub handle?)

Reply:
His GitHub handle should be [ac3eb](https://github.com/ac3eb) but not sure if he's part of the org.

Reply:
yea, I couldn't find his handle..so sent on [slack](https://broadinstitute.slack.com/archives/GB1VAM14J/p1630593811142600) directly